warehouse/integrations/bigquery/bigquery.go

```diff
@@ -688,7 +716,9 @@ func (bq *BigQuery) LoadUserTables(ctx context.Context) (errorMap map[string]err
 
 	log.Infon("Loading data")
 	var partitionedUsersTable string
-	if bq.avoidPartitionDecorator() {
+
+	// avoid writing to partioned table if there is a no or custom partition instead of ingestion time partition i.e _PARTITIONTIME
+	if bq.avoidPartitionDecorator() || usersTablePartitionInWarehouse == nil || usersTablePartitionInWarehouse.Field != "" {
```

Comment:
Aren't the conditions `usersTablePartitionInWarehouse == nil` and `usersTablePartitionInWarehouse.Field != ""` effectively opposites? We're trying to check for the absence of a partition field, right?

Reply:
No

`usersTablePartitionInWarehouse == nil` indicates if table has no partition.  If table has no partition we should avoid loading into partitioned tables.

`usersTablePartitionInWarehouse.Field != ""`  indicates there is a custom partition column set other than ingestion time. we should avoid loading into partition in this case

Reply:
If the partition column is ingestion time, is `usersTablePartitionInWarehouse == nil` true and `usersTablePartitionInWarehouse.Field != ""` false ?